packages/devtools_app/lib/src/screens/inspector_v2/widget_properties/properties_view.dart

```diff
@@ -80,14 +92,225 @@ class _DetailsTableState extends State<DetailsTable> {
+                                child: propertiesList,
+                              ),
+                    ],
+                  ),
+                );
+              }
+
+              return PropertyItem(
+                index: index - layoutExplorerOffset,
+                properties: properties,
+              );
+            },
+          ),
+        );
+      },
+    );
+  }
+}
+
+/// List of the widget's layout properties.
+class LayoutPropertiesList extends StatelessWidget {
+  const LayoutPropertiesList({
+    super.key,
+    required this.widgetHeights,
+    required this.widgetWidths,
+  });
+
+  final WidgetSizes? widgetHeights;
+  final WidgetSizes? widgetWidths;
+
+  LayoutWidthsAndHeights? get widthsAndHeights =>
+      widgetHeights != null && widgetWidths != null
+          ? LayoutWidthsAndHeights(
+              widths: widgetWidths!,
+              heights: widgetHeights!,
+            )
+          : null;
+
+  @override
+  Widget build(BuildContext context) {
+    if (widthsAndHeights == null) return const SizedBox.shrink();
+
+    final LayoutWidthsAndHeights(
+      :widgetHeight,
+      :widgetWidth,
+      :topPadding,
+      :bottomPadding,
+      :leftPadding,
+      :rightPadding,
+      :hasTopPadding,
+      :hasBottomPadding,
+      :hasLeftPadding,
+      :hasRightPadding,
+    ) = widthsAndHeights!;
+
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        LayoutPropertyItem(
+          name: 'height',
+          value: widgetHeight,
+        ),
+        LayoutPropertyItem(
+          name: 'width',
+          value: widgetWidth,
+        ),
+        if (hasTopPadding)
+          LayoutPropertyItem(
+            name: 'top padding',
+            value: topPadding,
+          ),
+        if (hasBottomPadding)
+          LayoutPropertyItem(
+            name: 'bottom padding',
+            value: bottomPadding,
+          ),
+        if (hasLeftPadding)
+          LayoutPropertyItem(
+            name: 'left padding',
+            value: leftPadding,
+          ),
+        if (hasRightPadding)
+          LayoutPropertyItem(
+            name: 'right padding',
+            value: rightPadding,
+          ),
+      ],
+    );
+  }
+}
+
+/// A layout property's name and value displayed in the [LayoutPropertiesList].
+class LayoutPropertyItem extends StatelessWidget {
+  const LayoutPropertyItem({
+    super.key,
+    required this.name,
+    required this.value,
+  });
+
+  final String name;
+  final double value;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(densePadding),
+      child: RichText(
+        text: TextSpan(
+          text: '$name: ',
+          style: Theme.of(context).subtleTextStyle,
+          children: [
+            TextSpan(
+              text: toStringAsFixed(value),
+              style: Theme.of(context).fixedFontStyle,
```

Comment:
put Theme.of(context) into a var at the top of this build method for shared use

Reply:
done!